include/logging/log_output.h

```diff
@@ -121,6 +130,18 @@ void log_output_msg_process(const struct log_output *output,
 void log_output_msg2_process(const struct log_output *log_output,
 			     struct log_msg2 *msg, uint32_t flags);
 
+/** @brief Process log messages v2 to SYS-T format.
+ *
+ * Function is using provided context with the buffer and output function to
+ * process formatted string and output the data in sys-t log output format.
+ *
+ * @param log_output Pointer to the log output instance.
+ * @param msg Log message.
+ * @param flag Optional flags.
+ */
+extern void log_output_msg2_syst_process(const struct log_output *log_output,
```

Comment:
There is no need for `extern` here.

Reply:
Makes sense. Removed the extern keyword. Thanks for pointing it out.